client/src/containers/PostDashboard.tsx

```diff
@@ -9,16 +9,31 @@ import Header from '../components/Header';
 import PostPreview from '../components/PostPreview';
 import CreatePost from '../components/CreatePost';
 
+import Select from '@mui/material/Select';
+import MenuItem from '@mui/material/MenuItem';
+
 import ServerApi, { PostUserPreview } from '../api/v1';
 
 export const POSTS_PER_PAGE = 6; // Maximum (previewable) posts per page.
 
+const postTypes = [
+  'All',
+  'Events',
+  'Clubs',
+  'Textbooks',
+  'Roommates',
+  'Opportunities',
+];
+
```

Comment:
The types should really only exist in one spot. `constants.ts`? Helps with misspellings and such